templates/README.tpl.md

```diff
@@ -0,0 +1,166 @@
+
+You can then clone the app repository and install dependencies:
+
+```sh
+$ git clone https://github.com/cozy/<SLUG_GH>.git
+$ cd <SLUG_GH>
+$ npm install
+```
+
+:pushpin: If you use a node environment wrapper like [nvm] or [ndenv], don't forget to set your local node version before doing a `npm install`.
+
+Cozy's apps use a standard set of _npm scripts_ to run common tasks. You can so start you development workflow with:
+
+```sh
+$ cd <SLUG_GH>
+$ npm run watch
+```
+
+and point your browser to http://localhost:<APP_PORT>.
+
+
+### Run it inside the VM
+
+You can easily view your current running app in your VM, use [cozy-dev]:
+
+```sh
+# in a terminal, run your app in watch mode
+$ cd <SLUG_GH>
+$ npm run watch
+```
+
+```sh
+# in another terminal, install cozy-dev (first time) and run the deploy
+$ cd <SLUG_GH>
+$ npm install -g cozy-dev
+$ cozy-dev deploy <APP_PORT>
+```
+
+and point your browser to your dev VM at http://localhost:9104, your app is now available in your dashboard.
+
+
+### Tests
+
+Tests are run by [mocha] under the hood, and written using [chai] and [sinon]. You can easily run the tests suite with:
+
+```sh
+$ cd <SLUG_GH>
+$ npm run test
+```
+
+:pushpin: Don't forget to update / create new tests when you contribute to code to keep the app the consistent.
+
+
+### Resources
+
+All documentation is located in the `/docs` app directory. It provides an exhaustive documentation about worflows (installation, development, pull-requests…), architecture, code consistency, data structures, dependencies, and more.
+
+Feel free to read it and fix / update it if needed, all comments and feedback to improve it are welcome!
+
+
+### Open a Pull-Request
+
+If you want to work on <APP_NAME> and submit code modifications, feel free to open pull-requests! Our basic worflow is:
+
+- PR point to the `development` branch
+- You need to cover your code and feature by tests
+- You may add documentation in the `/docs` directory to explain your choices if needed
+- you do _not_ need to build app to submit a PR
+- you should update the Transifex source locale file if you modify it for your feature needs (see Localization section below)
+
+
+Community
+---------
+
+### Localization
+
+Localization and translations are handled by [Transifex][tx], which is used by all Cozy's apps.
+
+As a _translator_, you can login to [Transifex][tx-signin] (using your Github account) and claim an access to the [app repository][tx-app]. Locales are pulled when app is build before publishing.
+
+As a _developer_, you must [configure the transifex client][tx-client], and claim an access as _maintainer_ is the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push` command.
+
+
+### Maintainer
+
+The lead maintainer for Cozy <APP_NAME> is <APP_MAINTAINER>, send him/her a :beers: to say hello!
```

Comment:
Nope, they're just vars to replace in the template, they're not supposed to stay in final README file, so I use `<>` as a delimiter to find/replace them quickly 😃